ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityCompositeAnswer.component.ts

```diff
@@ -118,6 +116,7 @@ export class ActivityCompositeAnswer implements OnChanges {
         newQuestionBlock.validators.forEach((validator) => validator.question = newQuestionBlock);
         newQuestionBlock.answer = answerContainer === null ? null : answerContainer.value;
         newQuestionBlock.shown = shown;
+        newQuestionBlock.question = '';
```

Comment:
These changes related to this bug:
![image](https://user-images.githubusercontent.com/30126907/79125686-3925a400-7da7-11ea-91a8-7081049f7350.png)
On the frontend, we shouldn't show 'Date' question, but the mobile client requires that field.
I see two solutions:
1. I looked through all composite blocks on our websites and look like composite blocks show only the main question, sub-questions don't show his own question, so we can just clear question text(like in my PR)
2. We can add `hidePrompt` field in the question to control it

What do you think @zyme, @MocanaAtBroad, @yufengwng?